source/manual/run-ab-test.html.md

```diff
@@ -69,6 +69,11 @@ need more data.
 1. Use the [govuk_ab_testing gem][govuk_ab_testing] to serve different versions to your users. It can be configured with the analytics dimension selected in step 2.
 1. To activate or deactivate the test, or to change the B percentage, update your test in [the govuk-cdn-config repo][govuk-cdn-config] and deploy the dictionaries.
 
+If you're making a change to Search API, you may also want to test using
+the [search performance explorer](https://github.com/alphagov/search-performance-explorer/).
+In which case, you'll need to [add your AB test](https://github.com/alphagov/search-performance-explorer/commit/01e3d21ceca96951425b5ddc87116f0756411691) to the application, and manually
+deploy it to Heroku.
```

Comment:
Picky, feel free to disregard - can this line break be removed and the sentence joined to the previous line?

Reply:
The line break isn't visible in the rendered version: https://github.com/alphagov/govuk-developer-docs/blob/59c79c3a4326ce8bc07a51dd28b6bbbf838a94f6/source/manual/run-ab-test.html.md I'll leave it for now :)